src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -331,3 +331,9 @@
 {% macro claimRejectedFreeMediationNOWithoutDQsForDefendantDashboard() %}
   {{ t('You’ve rejected the claim and suggested mediation. We’ll ask the claimant if they agree to take part in mediation.') }}
 {% endmacro %}
+
+
+
+
+
+
```

Comment:
can we please remove these additional empty lines.

Reply:
done